vispy/visuals/tests/test_image.py

```diff
@@ -337,9 +336,9 @@ def test_image_interpolation():
         image.interpolation = 'custom'
         image.custom_kernel = np.array([[0]])  # no sampling
         render = c.render()
-        assert np.allclose(render[left], white)
-        assert np.allclose(render[right], white)
-        assert np.allclose(render[center], white)
+        assert np.allclose(render[left], black)
+        assert np.allclose(render[right], black)
+        assert np.allclose(render[center], black)
```

Comment:
Now this makes sense. Once again, looking back at tests makes me wonder hwo I didn't notice this the first time around ^^' This is supposed to be black because, indeed, we are always sampling with a kernel that multiples by 0.